lib/kb/fake/bounded_context/pet_family/pet_parents.rb

```diff
@@ -26,6 +26,18 @@ def petparents_filterable_attributes
           json_response 200, contracts
         end
 
+        get '/v1/petparents/:key/referrals' do
+          json_response 200, referrals_by_pet_parent_key(params['key'])
+        end
+
+        post '/v1/petparents/:key/referrals' do
+          resource = JSON.parse(request.body.read)
+          resource = resource.merge 'key' => SecureRandom.uuid
+          resource = resource.merge 'petParentKey' => params['key']
```

Comment:
Is this attribute going to be present in the "real" response from kb? Or is it added here only for the purposes of filtering in `referrals_by_pet_parent_key` method?

I'm thinking that if it's going to be present in the real response - we could add it to the list of `attributes` of `Referral`.